routes/api/individualsearchresult.js

```diff
@@ -0,0 +1,25 @@
+var express = require('express');
+var router = express.Router();
+var Hotel = require('../../models/Hotel');
+var Booking = require('../../models/booking');
+
+router.get('/individual-search-result/:id', (req,res) =>{
+    var checkin = [];
+    var checkout = [];
+    Hotel.findById(req.params.id)
```

Comment:
Since you're getting the Hotel's ID from the URL string instead of from a JSON object, you access it by calling `req.query.id` instead so it would be:
`Hotel.findById(req.query.id)` as said by @HongbinZheng